client/src/lit/compat/lazy-bcd-table.js

```diff
@@ -0,0 +1,85 @@
+import { LitElement, html } from "lit";
+import { createComponent } from "@lit/react";
+import React from "react";
+import { BCD_BASE_URL } from "../../env.ts";
+import "./bcd-table.js";
+
+class LazyBcdTable extends LitElement {
```

Comment:
nit:
```suggestion
class LazyBCDTable extends LitElement {
```

Reply:
Renamed to `LazyCompatTable` in 3d1fcac9efea7bfc48120545414fd30fa5c28941.